processing/src/main/java/io/druid/query/groupby/epinephelinae/ConcurrentGrouper.java

```diff
@@ -94,6 +96,8 @@ public ConcurrentGrouper(
           )
       );
     }
+
+    this.entryComparator = keySerdeFactory.factorize().entryComparator();
```

Comment:
Factorizing and throwing away an unused keySerde here seems wrong; could this be a method on KeySerdeFactory instead? It could be on both KeySerdeFactory and on KeySerde, if that makes life easier.